CMakeLists.txt

```diff
@@ -12,10 +14,10 @@ set(CXX_STANDARD_REQUIRED ON)
 # Set the version number
 set (FSTL_VERSION_MAJOR "0")
 set (FSTL_VERSION_MINOR "9")
-set (FSTL_VERSION_PATCH "4")
+set (FSTL_VERSION_PATCH "5-snapshot")
```

Comment:
I'm on the fence about including `snapshot` in the version patch here but I'll let @mkeeter weigh in as well.

Reply:
My reasoning is that this project doesn't have enough active development for numbered subreleases (a la alpha1.5), but people are referencing the version number and/or packaging for different systems. Having multiple states with identical (and non-snapshot) version numbers could potentially cause problems for those people.
  But, of course, I am fine with whatever you guys decide on 👍

Reply:
I see; from that perspective I think it makes sense in that context.